### ISSUE TITLE
Tried to call 'CompactRaidFrame1:Show()'

### ISSUE THREAD

**Issue by Uruloki84** [SLOKnightfall/BetterWardrobe#110]
I'm not an expert, but could "Can I Mog it?" or "Handy Notes" be an issue? Since this addon appears to modify the tooltip; however, the bug is with the blizzard compact raid frames, which I have not touched with an addon. Does the author know of any known conflicting addons? 
-----
17x [ADDON_ACTION_BLOCKED] AddOn 'BetterWardrobe' tried to call the protected function 'CompactRaidFrame1:Show()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `Show'
[string "@FrameXML\CompactUnitFrame.lua"]:375: in function `CompactUnitFrame_UpdateVisible'
[string "@FrameXML\CompactUnitFrame.lua"]:306: in function `CompactUnitFrame_UpdateAll'
[string "@FrameXML\CompactUnitFrame.lua"]:125: in function <FrameXML\CompactUnitFrame.lua:57>

**Comment by SLOKnightfall** [comment]
It it caused by taint from using the default Blizzard UI.  Could you ALT-Click on the bug grabber minimap icon an see if BetterWardrobe is still being listed.  I did a rework of the menus that should have fixed the issue  of this addon generating taint but Buggrabber/Bugsack likes to reuse similar errors so a different addon could be the one generating it now but it is just getting tacked on to an old error listing this one.

**Comment by bellexandria** [comment]
Hi, I just received this:

```2x [ADDON_ACTION_BLOCKED] AddOn 'BetterWardrobe' tried to call the protected function 'CompactPartyFrame:unusedFunc()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `unusedFunc'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:160: in function `CompactRaidFrameContainer_LayoutFrames'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:130: in function `CompactRaidFrameContainer_TryUpdate'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:62: in function `CompactRaidFrameContainer_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>
```

**Comment by bellexandria** [comment]
Also this one last night:

```
1x [ADDON_ACTION_BLOCKED] AddOn 'BetterWardrobe' tried to call the protected function 'CompactPartyFrameMember1:Show()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `Show'
[string "@FrameXML\CompactUnitFrame.lua"]:375: in function `CompactUnitFrame_UpdateVisible'
[string "@FrameXML\CompactUnitFrame.lua"]:306: in function `CompactUnitFrame_UpdateAll'
[string "@FrameXML\CompactUnitFrame.lua"]:154: in function <FrameXML\CompactUnitFrame.lua:57>
```

**Comment by Uruloki84** [comment]
I cleared all the bugs from bug graggers like you asked. However, the error popped back up. I was on a taming spree on my hunter in WoD and noticed that it usually happened when I tamed a new pet and as soon as the tame cast finished and the pet leveled up to match it would throw the error.
I just confirmed on another hunter that taming a rare that maintains its name after tamed causes the error to come up. I then immediately tamed a normal creature and the error did not appear. Maybe this helps or maybe it tells us it's not BetterWardrobe.
----------
2x [ADDON_ACTION_BLOCKED] AddOn 'BetterWardrobe' tried to call the protected function 'CompactRaidFrame2:SetAttribute()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML\CompactUnitFrame.lua"]:178: in function `CompactUnitFrame_SetUnit'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:318: in function `CompactRaidFrameContainer_AddUnitFrame'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:272: in function `CompactRaidFrameContainer_AddPets'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:182: in function `CompactRaidFrameContainer_LayoutFrames'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:130: in function `CompactRaidFrameContainer_TryUpdate'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:62: in function `CompactRaidFrameContainer_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>
